packages/page-experience/lib/PageDataGatherer.js

```diff
@@ -114,6 +114,23 @@ class PageAnalyzer {
         return css;
       };
 
+      /**
```

Comment:
this should go into linter as it's a static check

Reply:
Would that be different from [the current check](https://github.com/ampproject/amp-toolbox/blob/main/packages/linter/src/rules/ModuleRuntimeUsed.ts)?

Reply:
No